tools/mpremote/mpremote/transport_serial.py

```diff
@@ -747,7 +752,7 @@ def __mount():
 
 # Apply basic compression on hook code.
 for key, value in fs_hook_cmds.items():
-    fs_hook_code = re.sub(key, str(value), fs_hook_code)
+    fs_hook_code = re.sub(rf'(\W){key}(\W)', rf"\g<1>{value}\g<2>", fs_hook_code)
```

Comment:
How about replacing the entire loop with a single line:
```py
fs_hook_code = re.sub(r"CMD_[A-Z]+", lambda match: fs_hook_cmds[match.group(0)], fs_hook_code)
```

Reply:
I like it. Minor tweaks to make it work and it tests out fine with the script I used earlier. Pushed now.

Reply:
Very good!